.pyup.yml

```diff
@@ -7,5 +7,8 @@ label_prs: update
 assignees: sfarrens
 requirements:
   - requirements.txt
+      pin: False
```

Comment:
Whats the use of this arg? I mean i am fine having it, just wondering where it will be used

Reply:
If I have correctly interpreted the [pyup docs](https://pyup.io/docs/bot/config/), this should prevent the pyup bot from creating PRs to pin the requirements in this file. I would like to keep this feature for the documentation, hence I have this set this option independently for each file.